js/apps/admin-ui/src/user/CreateUser.tsx

```diff
@@ -66,15 +94,15 @@ export default function CreateUser() {
       />
       <PageSection variant="light" className="pf-u-p-0">
         <UserProfileProvider>
```

Comment:
guess we don't need this `UserProfileProvider` here anymore

Reply:
Yeah, good catch. I'll remove it.

Reply:
Done